geopandas/array.py

```diff
@@ -301,11 +301,8 @@ def sindex_generated(self):
 
         Examples
         --------
-        Checking if the spatial index has been generated.
+        Given a GeoDataFrame 'gdf':
 
-        >>> from shapely.geometry import Point
-        >>> d = {'col1': ['name1', 'name2'], 'geometry': [Point(1, 2), Point(2, 1)]}
-        >>> gdf = geopandas.GeoDataFrame(d, crs="EPSG:4326")
```

Comment:
Sorry to come late, but can you revert it to the previous option where you actually created `gdf`? After #1651 we now test docstring examples and this would have to be either ignored or reverted.

Reply:
No worries. I just reverted. Does it also need `import geopandas`?

Reply:
No, that is imported globally, this is okay. Thanks.